metadefender-plugin-server/src/main/scala/com/opswat/teamcity/ArtifactScanner.scala

```diff
@@ -314,6 +348,7 @@ class ArtifactScanner(config: MConfigManager) extends BuildServerAdapter {
                         infectedFiles = tm :: infectedFiles
                       }
                     } else if (processResult.toLowerCase.equals("allowed")) {
+                      reportInfo(tm)

```

Comment:
I wouldn't change the default output behavior as it has the possibility to spam the build log. We could add a config for this

Reply:
I think this change is always necessary if sandbox is enabled. Consider the case when multiscanning says the file is clean, but the sandbox returns an infected verdict. If we don't print this line, then the user will not know where to check the complete scan results.
Btw, the TC build logs are already enormous, so "spamming" is not really a concern ;)